### ISSUE TITLE
[Screen reader - WIX - Setup Progress] Narrator does not provide information about setup progress while installing the setup.

### ISSUE THREAD

**Issue by tgugler** [wixtoolset/issues#6224]
If this issue is a bug:

* Which version of WiX are you building with?

> 3.14.0.1119

* Which version of Visual Studio are you building with (if any)?

> Visual Studio 2017

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> 1.0.0.4

* Which version of .NET are you building with?

> Not applicable

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> Not applicable

* Describe the problem and the steps to reproduce it.

User Experience:
Screen reader user would face difficulty to get information about the setup progress if screen reader does not read it.

Pre Requisite:
Download exe file from https://www.microsoft.com/en-us/download/details.aspx?id=54688 ​
​
Repro Steps:​
1. Launch the "NPS Extension For Azure MFA" setup file.​
2. Navigate to "MICROSOFT SOFTWARE LICENSE TERMS" popup window and select "Install" button and launch the screen reader.
3. Verify the issue.

Actual Result:
Narrator does not provide information about setup progress while installing the setup.

Expected Result:
Narrator should provide information about setup progress while installing the setup.

Notes:
1. Same issue exists with JAWS.
2. Narrator does not provide any information about completion of setup installation.
3. Same issue exists for 'setup progress' pop-up window appears after selecting 'Modify->repair' the setup.

[MAS1.3.1_setup progress.wmv.zip](https://github.com/wixtoolset/issues/files/5043064/MAS1.3.1_setup.progress.wmv.zip)

**Comment by barnson** [comment]
What is it supposed to do? Please provide an example.
